--- bcel-5.1/src/java/org/apache/bcel/verifier/statics/Pass2Verifier.java.orig	2005-05-07 15:10:31.262642120 +0200
+++ bcel-5.1/src/java/org/apache/bcel/verifier/statics/Pass2Verifier.java	2005-05-07 15:10:22.642952512 +0200
@@ -60,6 +60,7 @@
 import org.apache.bcel.classfile.DescendingVisitor; // Use _this_ one!
 import org.apache.bcel.classfile.EmptyVisitor; // Use _this_ one!
 import org.apache.bcel.classfile.Visitor; // Use _this_ one!
+import org.apache.bcel.classfile.Deprecated; // Use _this_ one!
 import org.apache.bcel.generic.*;
 import org.apache.bcel.verifier.*;
 import org.apache.bcel.verifier.exc.*;
--- bcel-5.1/src/java/org/apache/bcel/verifier/statics/StringRepresentation.java.orig	2003-04-25 09:06:16.000000000 +0200
+++ bcel-5.1/src/java/org/apache/bcel/verifier/statics/StringRepresentation.java	2005-05-07 15:11:18.312489456 +0200
@@ -55,6 +55,7 @@
  */
 
 import org.apache.bcel.classfile.*;
+import org.apache.bcel.classfile.Deprecated; // use _this_ one!
 import org.apache.bcel.verifier.exc.*;
 
 /**
